src/main.c

```diff
@@ -643,6 +657,13 @@ int main(int argc, char *argv[])
 	parse_arguments(argc, argv);
 	srand(time(NULL));
 
+	/* Not cairo specific, so i think this is appropriate to put here */
```

Comment:
Not sure what this comment is about :) Can be removed.

Reply:
Was more about how i didn't know if you wanted it there or not.